### DISCUSSION TITLE
Redis 6.x support

### DISCUSSION THREAD

**Discussion by mcatanzariti** [dahomey-technologies/rustis#23] [General]
At first, rustis was developed for Redis 7.x.
I just committed a change to support cluster for Redis 6.x but some commands may not be supported correctly for Redis 6.x in standalone mode or cluster mode.

@mstyura Could you let me know any issue you could get running rustis on Redis 6.x?

Thank you!

**Comment by yyaroshevich** [comment]
Hello, @mcatanzariti!

I apologize for the delay in my response. I wasn't able to examine the compatibility with version 6.x earlier. However, I have now conducted the necessary tests, and it appears that `rustis` is functioning smoothly with Redis 6.2.12 in the application I'm currently working on. I sincerely appreciate your efforts in incorporating support for Redis clusters that are below version 7. Thank you very much.
